src/pages/Search/useSearch.ts

```diff
@@ -0,0 +1,22 @@
+import { SetStateAction, useState } from 'react';
+
+export default function useSearch(state : 'search' | 'trending' | null) {
+  const [text, setText] = useState('');
+  const [mode, setMode] = useState(state);
+
+  const textHandler = (e: { target: { value: SetStateAction<string> } }) => {
```

Comment:
```suggestion
  const textHandler:ChangeEventHandler<HTMLInputElement> = (e) => {
  const textHandler = (e: ChangeEvent<HTMLInputElement>) => {
```
두 방식 중 하나로 타입을 지정하는 것이 더 적절합니다.

Reply:
수정했습니다 [9707cf6](https://github.com/BCSDLab/JJBAKSA_FRONT_END/pull/18/commits/9707cf670d90309fcb8e359ec926fd121cbabca4)